sched/scheduler/stateful_scheduler.go

```diff
@@ -44,19 +44,22 @@ const DefaultDefaultTaskTimeout = 30 * time.Minute
 const DefaultTaskTimeoutOverhead = 15 * time.Second
 
 // Number of different requestors that can run jobs at any given time.
-const DefaultMaxRequestors = 100
+const DefaultMaxRequestors = 10
 
-// Number of jobs any single requestor can have.
+// Number of jobs any single requestor can have (to prevent spamming, not for scheduler fairness).
 const DefaultMaxJobsPerRequestor = 100
 
 // Expected number of nodes. Should roughly correspond with the actual number of healthy nodes.
```

Comment:
I don't understand this comment - there are thousands of nodes (hopefully most are healthy)... is this number of healthy nodes for a job?

Reply:
I don't like having too many knobs and this one is here primarily to address cluster size during worker redeploys. Scheduling will be suboptimal during this period but it's likely fine to get rid of this and use the actual cluster size directly.